cluster-network-setup.yml

```diff
@@ -1,6 +1,20 @@
 ---
 - hosts: localhost
-  gather_facts: no
+  gather_facts: False
+  vars:
+    cluster_name: swarm-bharat
+    cluster_user: fedora
```

Comment:
Let's pull some config like this out of the playbook. You can use a `group_vars/all` file, or an extra variables file (specify via `-e <file>`.

Playbooks should be general, roles even more so.